src/main/java/mekanism/common/config/GearConfig.java

```diff
@@ -21,8 +21,9 @@
     private static final String ENERGY_TABLET_CATEGORY = "energy_tablet";
     private static final String FLAMETHROWER_CATEGORY = "flamethrower";
     private static final String FREE_RUNNER_CATEGORY = "free_runner";
+    private static final String ARMORED_FREE_RUNNER_SUBCATEGORY = "armored_free_runner";
     private static final String JETPACK_CATEGORY = "jetpack";
-    private static final String ARMORED_JETPACK_SUBCATEGORY = "armored";
+    private static final String ARMORED_JETPACK_SUBCATEGORY = "armored_jetpack";
```

Comment:
This doesn't need to be changed as it is a subcategory to the jetpack category already so it basically is currently `jetpack.armored`

Reply:
Though I definitely think it may be worth renaming the constant to `ARMORED_SUBCATEGORY` and then just using it for both the free runner one and the jetpack one.

Reply:
Ah, right. Fixed.